server/checks/CommitMessage.js

```diff
@@ -0,0 +1,102 @@
+import Check from './Check'
+import { logger } from '../../common/debug'
+import { getIn } from '../../common/util'
+import * as EVENTS from '../model/GithubEvents'
+
+const CHECK_TYPE = 'commitmessage'
+const info = logger(CHECK_TYPE, 'info')
+const error = logger(CHECK_TYPE, 'error')
+const context = 'zappr-commitmessage'
```

Comment:
Could we maybe rename this to `zappr/commitmessage` and the context of the Approval Check to `zappr/approval`?